docker-compose.base.yml

```diff
@@ -79,7 +79,7 @@ services:
         # Note: please keep the default version in sync across
         #       `posthog` and the `charts-clickhouse` repos
         #
-        image: ${CLICKHOUSE_SERVER_IMAGE:-clickhouse/clickhouse-server:23.12.6.19-alpine}
+        image: ${CLICKHOUSE_SERVER_IMAGE:-clickhouse/clickhouse-server:23.12.6.19}
```

Comment:
This is probably a good idea moving off of `alpine` since this will reflect more what production is

Reply:
Yeah for sure -  in this case, a glibc binary is more than 2x faster than a MUSL libc binary - apparently the musl libc binary has a slow memory allocator (going for simplicity and portability over speed), but that's huge lol

Reply:
I actually started wondering if clickhouse would be faster locally running on a normal container vs an alpine one